src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -0,0 +1,431 @@
+# Electroencephalography (EEG)
+
+Support for EEG was developed as a [BIDS Extension Proposal](../06-extensions.md#bids-extension-proposals).
+Please cite the following paper when referring to this part of the standard in
+context of the academic literature:
+
+> Pernet, C. R., Appelhoff, S., Flandin, G., Phillips, C., Delorme, A., &
+> Oostenveld, R. (2018, December 6). BIDS-EEG: an extension to the Brain
+> Imaging Data Structure (BIDS) Specification for electroencephalography.
+> [https://doi.org/10.31234/osf.io/63a4y](https://doi.org/10.31234/osf.io/63a4y)
+
+The following example EEG datasets have been formatted using this specification
+and can be used for practical guidance when curating a new dataset.
+
+-   Single session per subject: [`eeg_matchingpennies`](https://doi.org/10.17605/OSF.IO/CJ2DR)
+-   Multiple sessions per subject: [`eeg_rishikesh`](https://doi.org/10.5281/zenodo.1490922)
+-   Combined with fMRI: [`eeg_rest_fmri`](https://osf.io/94c5t/files/)
+
+Further datasets are available from the [BIDS examples repository](https://github.com/bids-standard/bids-examples).
+
+## EEG recording data
+
+Template:
+
+```Text
+sub-<label>/
+    [ses-<label>]/
+      eeg/
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_eeg.<manufacturer_specific_extension>
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_eeg.json
+```
+
+The EEG community uses a variety of formats for storing raw data, and there is
+no single standard that all researchers agree on. For BIDS, EEG data MUST be
+stored in one of the following formats:
+
+-   [European data format](https://www.edfplus.info/) (`.edf`)
+
+-   [BrainVision data format](https://www.brainproducts.com/productdetails.php?id=21&tab=5)
+    (`.vhdr`, `.vmrk`, `.eeg`) by Brain Products GmbH
+
+-   The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab)
+    (`.set` and `.fdt` files)
+
+-   [Biosemi](https://www.biosemi.com/) data format (`.bdf`)
+
+It is RECOMMENDED to use the European data format, or the BrainVision data
```

Comment:
It is RECOMMENDED to use the European data format, or the BrainVision data
format. It is furthermore discouraged to use the other accepted formats over
these RECOMMENDED formats, particularly because there are conversion scripts
available in most commonly used programming languages to convert data into the
RECOMMENDED formats.